config/jobs/kubernetes/sig-network/sig-network-kind.yaml

```diff
@@ -1,10 +1,12 @@
 presubmits:
   kubernetes/kubernetes:
-  - name: pull-kubernetes-e2e-kind-dual-canary
+  - name: pull-kubernetes-e2e-kind-dual
     optional: true
-    always_run: false
-    skip_report: false
+    always_run: true
+    skip_report: true
```

Comment:
Three always run kind jobs is a lot. Can we replace the other two with this? Or at least one of them? We should definitely not skip reporting unless the intent is to gain confidence to replace

Reply:
this is to gain confidence ...
we need the ipv6 only because is the only job that gates on ipv6 ...
the dual is a superset of the ipv4, and we have all the other jobs for ipv4 ... 
I'd like to hear from Ben too